### ISSUE TITLE
Assessment Results

### ISSUE THREAD

**Issue by damwhit** [nwgambee/mod_0_skills#1]
@nwgambee nice work on this. You are officially `technical ready`! I have some food for thought, however: The only attribute I think I could do without is the atCapacity attribute. Why is that? Thinking back to the questions we ask to tell whether or not something is a good attribute.

**Comment by nwgambee** [comment]
Hi David, thanks! And I see my mistake there, `atCapacity` is not a piece of data that could stay the same over the course of the object's lifetime. Thanks for pointing that out.

**Comment by damwhit** [comment]
True, but I was getting at the fact that `atCapacity` relies on underlying data so it would be better served as a method
